packages/fxa-settings/src/components/Settings/index.tsx

```diff
@@ -34,7 +37,7 @@ export const Settings = ({ account }: { account: Account }) => {
             header="Password"
             headerValueClassName="tracking-wider"
             headerValue="••••••••••••••••••"
-            route="#"
+            route="/beta/settings/change_password"
```

Comment:
I'm [currently discussing](https://github.com/mozilla/fxa/pull/6187#discussion_r468886110) this with @LZoog, but I'd like us to consider constantizing paths. Would love your input!

Reply:
I was going to take a look at doing that in https://jira.mozilla.com/browse/FXA-1705

Reply:
Works for me!